plenum/server/request_handlers/txn_author_agreement_handler.py

```diff
@@ -57,3 +58,10 @@ def update_state(self, txn, prev_result, request, is_committed=False):
 
     def authorize(self, request):
         StaticTAAHelper.authorize(self.database_manager, request)
+
+    def _decode_state_value(self, encoded):
```

Comment:
Why do we need to override it? It looks like `_decode_state_value` in `WriteRequestHandler` does the same.

Reply:
In WriteRequestHandler  we use ```domain_state_serializer```, but there use ```config_state_serializer```.